tests/src/system/basic/CLIActionTests.java

```diff
@@ -1,501 +0,0 @@
-            String expected = String.format("The message '%s' has", now);
-            boolean present = wsk.logsForActivationContain(activationId, expected, DEFAULT_WAIT);
-            assertTrue("Expected '" + expected + "' which is missing in log for activation " + activationId, present);
-            assertTrue("Result should have number of words", result.contains(String.valueOf(numWords)));
-            assertTrue("Expected result to not contain error", !result.contains("error"));
-        } finally {
-            wsk.delete(Action, action);
-        }
-    }
-
-    @Test(timeout=120*1000)
-    public void invokeNestedBlockingAction() throws Exception {
-        String parentAction = "nestedBlockingAction";
-        String childAction = "wc";
-        try {
-            wsk.sanitize(Action, childAction);
-            wsk.sanitize(Action, parentAction);
-            wsk.createAction(childAction, TestUtils.getCatalogFilename("samples/wc.js"));
-            wsk.createAction(parentAction, TestUtils.getCatalogFilename("samples/wcbin.js"), 120*1000);
-            String now = String.format("%s %s", parentAction, new Date().toString());
-            int numWords = now.trim().split("\\s").length;
-            String numWordsBinary = Integer.toBinaryString(numWords) + " (base 2)";
-            Pair<String, String> pair = wsk.invokeBlocking(parentAction, TestUtils.makeParameter("payload", now));
-            String activationId = pair.fst;
-            String result = pair.snd;
-            assertTrue("Result "+activationId+" should have number of words", result.contains(numWordsBinary));
-            assertTrue("Expected result to not contain error", !result.contains("error"));
-        } finally {
-            wsk.delete(Action, childAction);
-            wsk.delete(Action, parentAction);
-        }
-    }
-
-    @Test(timeout=120*1000)
-    public void copyAction() throws Exception {
-        String action = "copyAction";
-        try {
-            wsk.sanitize(Action, action);
-            wsk.copyAction(action, "/whisk.system/samples/wordCount");
-            String now = String.format("%s %s", action, new Date().toString());
-            String activationId = wsk.invoke(action, TestUtils.makeParameter("payload", now));
-            String expected = String.format("The message '%s' has", now);
-            boolean present = wsk.logsForActivationContain(activationId, expected, DEFAULT_WAIT);
-            assertTrue("Expected '" + expected + "' which is missing in log for activation " + activationId, present);
-            String result = wsk.getResult(activationId).stdout.trim();
-            assertTrue("Expected result of " + activationId + " to be non-empty", result.contains("7"));
-            assertTrue("Expected result to not contain error", !result.contains("error"));
-        } finally {
-            wsk.delete(Action, action);
-        }
-    }
-
-    @Test(timeout=120*1000)
-    public void updateAction() throws Exception {
```

Comment:
I think UpdateAction was testing that second invoke runs the updated action - if that's covered ok.